### ISSUE TITLE
Incorrect invalid opcode error

### ISSUE THREAD

**Issue by thisbejim** [ethereum/browser-solidity#633]
When forwarding ether from a function call in Remix it will throw an invalid opcode error.

Example:
```
pragma solidity ^0.4.8;

contract Example {

    function forward(address receiver) payable returns (bool) {
        if (!receiver.send(msg.value)) throw;
        return true; 
    }
}
```

Call the above function in Remix to send the msg.value amount on to another address and you will get the error.

I have confirmed this Example contract works as expected using truffle + testrpc.

**Comment by axic** [comment]
I think the reason is #634.

**Comment by yann300** [comment]
thanks @axic

**Comment by axic** [comment]
Fixed by #635.
